app/javascript/components/close_contact/CloseContact.js

```diff
@@ -31,6 +32,14 @@ class CloseContact extends React.Component {
       : 'enter additional information about monitoree’s potential exposure';
   }
 
+  formatPhoneNumber(phone) {
```

Comment:
Where is this being used here? I don't see it.

Reply:
sorry, I think I copied this over from another file that used it but it seems like it isn't necessary here, will remove